thunderbird/processes/wps_generate_prsn.py

```diff
@@ -10,12 +10,13 @@
 
 # Tool imports
 from dp.generate_prsn import generate_prsn_file
-from dp.generate_prsn import dry_run as dry_run_info
+from dp.generate_prsn import dry_run as dry_run_handler
```

Comment:
Is there a particular reason we are changing the name of the method?

Reply:
The reason is we already have an input argument named dry_run. Python recognized it as a boolean value when it was passed into dry_run_info.